### ISSUE TITLE
COCO number of classes

### ISSUE THREAD

**Issue by Bala93** [kazuto1011/deeplab-pytorch#112]
Why does COCO have 91 classes, compared to 80 classes ?

**Comment by kazuto1011** [comment]
COCO has labels for only 80 classes, while the indices range from 0 to 90 ([11 classes have no label](https://github.com/nightrome/cocostuff/blob/master/labels.md)).
To deal with this, the model simply outputs 91 class logits, and only 80 classes are supervised.
I understand that's kind of redundant, but I just followed [the official Caffe implementation](https://github.com/nightrome/cocostuff/blob/876a7db61f4957888b609011df25e039e512fb48/models/deeplab/run_cocostuff_resnet101.sh#L12) so that the pre-trained weights could be loaded as is.

**Comment by Bala93** [comment]
Thank you for the clarification.

**Comment by Bala93** [comment]
I would also like to know one more thing. How do I train for deeplab with 90 classes ? In the following file (https://github.com/kazuto1011/deeplab-pytorch/blob/master/libs/datasets/__init__.py), coco.yaml provision is not provided. I am working with train2014 dataset.

**Comment by Italy2006** [comment]
> I would also like to know one more thing. How do I train for deeplab with 90 classes ? In the following file (https://github.com/kazuto1011/deeplab-pytorch/blob/master/libs/datasets/__init__.py), coco.yaml provision is not provided. I am working with train2014 dataset.

Sorry but do you solve this problem? I have the same question as you too, thanks for your reply
